2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,36 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
```

Comment:
The recipe objects look good 👍 
But take a look at how they want you to output the recipe information on lines 13. For example, the output should look something like this:
```
  Mole
  Serves: 2
  Ingredients:
  cinnamon
  cumin
  cocoa
```